src/mode/new.rs

```diff
@@ -0,0 +1,52 @@
+use std::{process::Command, env, path::Path, fs::File, io::Write};
+
+use clap::Parser;
+use anyhow::{anyhow, Result};
+
+use crate::mode;
+
+#[derive(Parser, Debug)]
+#[command(version)]
+pub struct Args {
+    #[arg(index = 1)]
+    pub new_args: Vec<String>,
+}
+
+fn add_lunatic_main_file() {
+    let text = format!(
```

Comment:
Having it injected here makes maitenance difficult. a templating engine e.g. like https://crates.io/crates/askama could help.